src/main/kotlin/com/terraformation/backend/search/api/SearchController.kt

```diff
@@ -152,7 +154,9 @@ class SearchController(
 
   private fun resolvePrefix(prefix: String?): SearchFieldPrefix {
     val table =
-        if (prefix != null) {
+        if (nonOrganizationTables.containsKey(prefix)) {
+          nonOrganizationTables[prefix]!!
```

Comment:
The original error I was seeing before this change was "Sublist documents not found"